glusterd2/middleware/auth.go

```diff
@@ -31,11 +31,27 @@ func getAuthSecret(issuer string) string {
 	return ""
 }
 
+//isRestAuthRequired return true for few URL which doesn't require authentication
+func isRestAuthRequired(url string) bool {
```

Comment:
flip the return values. Func name and return values are confusing

Reply:
done